.gitattributes

```diff
@@ -0,0 +1,12 @@
+# Set the default behavior for line endings, in case people don't have core.autocrlf set.
```

Comment:
I recommend removing the vscode-induced changes and files: .gitattributes and .gitignore

Reply:
removed .gitattributes and changes made within .gitignore.